src/app/components/core/notifications/notifications.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="container is-fluid">
+  <div>
+    <p class="notification is-warning" *ngIf="!isNotificationServiceAvailable">
+      Notification service is not configured
+      and enabled. <a (click)="addNotificationService()"> Enable Now</a></p>
+    <p class="notification is-warning" *ngIf="isNotificationServiceAvailable && !isNotificationServiceEnabled">
+      Notification service is not enabled. <a (click)="enableNotificationService()"> Enable Now</a></p>
+  </div>
```

Comment:
We should also provide an option (message body info) to allow to `Disable Now` (to disable notification schedule) and show a warning alert to confirm .

Not sure if we should also have  option to `Delete` as well?!